source/docs/articles/organizations/change-management.md

```diff
@@ -54,43 +68,63 @@ Change Management is enabled for all Pantheon for Agencies Partners, and for som
 .tg .tg-e3zv{font-weight:bold}
 </style>
 <table class="tg">
-<tr>
-<th class="tg-e3zv">Role</th>
-<th class="tg-e3zv">Organization Dashboard</th>
-<th class="tg-e3zv">Site Dashboard</th>
-</tr>
-<tr>
-<td class="tg-031e">Owner</td>
-<td class="tg-031e">Functionally the same as an Organization Admin. This is the person/org who pays for the Site.</td>
-<td class="tg-031e">Same as Organization Administrator. Pantheon for Agencies sites are owned by the person who created them until a payment instrument is associated. All roles can "own" sites in this way.</td>
-</tr>
-<tr>
-<td class="tg-031e">Organization Administrator</td>
-<td class="tg-031e">- Manage Settings<br>- Remove Members<br>- Edit Member Roles<br>- Delete Sites<br>- Remove Sites from the Organization</td>
-<td class="tg-031e">- If the Site is owned by the Organization, the Organization Administrator functions as an Owner.<br>- If the Site is not owned by the Organization, the Organization Administrator functions as a Team Member.</td>
-</tr>
-<tr>
-<td class="tg-031e">Team Member</td>
-<td class="tg-031e">- Add Org Team members and developers (but cannot remove/edit roles)</td>
-<td class="tg-031e">- Manage Settings<br>- Manage Team (add/remove)<br>- Deploy Code to Test/Live</td>
-</tr>
-<tr>
-<td class="tg-031e">Developer</td>
-<td class="tg-031e">- Tag Sites<br>- Access Sites<br>- Create Sites</td>
-<td class="tg-031e">- Commit code to Multidev and Dev environments<br>- Cannot deploy code or clone content to Test/Live, unless invited to the site or were the creator of the site, in which case they are on the site team as member or owner.</td>
-</tr>
-<tr>
-<td class="tg-031e">Unprivileged*</td>
-<td class="tg-031e">Disabled
-<td class="tg-031e">- Create new Sites<br>- Cannot deploy to Test/Live</td>
-</tr>
+
+    <tr>
+        <th><strong>Role</strong></th>
+        <th><strong>Permissions</strong></th>
+    </tr>
+    <tr>
+        <td>Administrator</td>
+        <td>* Manage settings<br>
+        * Remove members    <br>
```

Comment:
I had a comment about using * for bullets. Can we end this and use `& #8226; or & bull;` - unicode bullets?